cluster-autoscaler/cloudprovider/gce/gce_manager.go

```diff
@@ -229,6 +247,79 @@ func (m *GceManager) RegisterMig(mig *Mig) bool {
 	return !updated
 }
 
+// UnregisterMig unregisters mig in Gce Manager. Returns true if the node group has been removed.
+func (m *GceManager) UnregisterMig(toBeRemoved *Mig) bool {
+	m.migsMutex.Lock()
+	defer m.migsMutex.Unlock()
+
+	newMigs := make([]*migInformation, 0, len(m.migs))
+	found := false
+	for _, mig := range m.migs {
+		if mig.config.GceRef == toBeRemoved.GceRef {
+			glog.V(4).Infof("Unregistered Mig %s/%s/%s", toBeRemoved.GceRef.Project, toBeRemoved.GceRef.Zone,
```

Comment:
V(0) or V(1). This should happen very rarely and it feels like a very important information.

Reply:
Done.